ehr/resources/reports/schemas/study/Pedigree/Pedigree.r

```diff
@@ -13,10 +13,10 @@ library(Rlabkey)
 #str(labkey.data);
 #warnings();
 
-labkey.acceptSelfSignedCerts();
+labkey.setCurlOptions(ssl_verifypeer = FALSE, ssl_verifyhost = FALSE)
 
 
-if ((length(labkey.data$id) == 0) | (is.na(labkey.data$dam) & is.na(labkey.data$sire))){
+if ((length(labkey.data$id) == 0) || (is.na(labkey.data$dam) & is.na(labkey.data$sire))){
```

Comment:
the original was probably wrong in multi-animal cases, but I might write this like:

if ((length(labkey.data$id) == 0) || all(is.na(labkey.data$dam)) & is.na(labkey.data$sire)))){
 
which would require all rows to lack dam+sire.

Reply:
Thanks for the updated snippet. Now updated!